server/src/test/java/org/elasticsearch/action/admin/indices/close/TransportVerifyShardBeforeCloseActionTests.java

```diff
@@ -166,11 +166,11 @@ public void testShardIsFlushed() throws Throwable {
     }
 
     public void testOperationFailsWithOnGoingOps() {
```

Comment:
Should we rename test to `testOperationFailsWhenNotBlocked`?

Reply:
Yes, we should, thanks. I pushed 1fa0434.